ramp4-pcar4/ramp4-pcar4
# Resolve additional wave errors and increased panel default width

### Related Item(s)
#2285, #2113 

### Changes
- Fixes an issue where large panels would overlap the mapnav button on smaller screen sizes
- Resolves the WAVE errors on Details panel hyperlinks

### Notes
Didn't fix the alt text error on the empty basemap image.  Samples that don't have the missing image don't have the error. 

### Testing

#2113 
Steps:
1. Open any sample with a legend and other fixtures
2. Open an additional panel
3. Open the grid
4. Resize the window slowly until its below 320 px (Min supported instance size)
5. Witness no overlap

#2285 
Steps:
1. Open the legacy default sample
2. Click on a purple or navy blue map point or open one of the "Releases of cadmium" grids and press a details icon
3. Verify that the WAVE tool is no longer grousin about the hyperlinks


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2359)
